src/client/users/store.js

```diff
@@ -5,6 +5,7 @@ import {actions as authActions} from '../auth/actions';
 function revive(state) {
   // Handle case user was authenticated on the server.
   const viewer = state && state.get('viewer');
+  User.isLoggedIn = !!viewer;
```

Comment:
It is a very bad idea to do this on server. You must be 100 % that there will be no context switch between invoking this and actually calling `willTransitionTo` on the component, otherwise next request will rewrite it for you. We've ran to big trouble with aproach like this.